backend/app/services/scheduler.py

```
@@ -0,0 +1,233 @@
+from app.services.custom_scraper import CustomScraper
+from app.services.news_fetcher import NewsFetcher
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class TaskScheduler:
+    def __init__(self):
+        self.running = False
+        self.db = SessionLocal()
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)